jpegenc/subblocks/RLE/RLECore/entropycoder.py

```diff
@@ -8,31 +8,31 @@ def two2bin(num):
     return inum + 1
 
 
-def nbits(num, width):
+def nbits(num, width_data):
```

Comment:
Understand now, you needed a convertible version, I still don't understand the last return though?